homeassistant/components/geniushub/strings.json

```diff
@@ -1,4 +1,40 @@
 {
+  "config": {
+    "step": {
+      "user": {
+        "title": "Genius Hub Configuration",
+        "menu_options": {
+          "option_1": "Option 1: Hub hostname/address with user credentials",
+          "option_2": "Option 2: Hub token only"
+        }
+      },
+      "option_1": {
+        "title": "Genius Hub Configuration Option 1",
+        "data": {
+          "host": "The hostname/IP address of the Genius Hub.",
+          "mac": "The MAC address of the Hub’s ethernet port.",
+          "password": "Your Genius Hub password.",
+          "username": "Your Genius Hub username."
```

Comment:
You can reference other translation keys here, checkout some other strings.json to see.

You can give the extra context in `data_description` (have a global search and check it out, its in strings.json files)